plenum/test/plugin/test_auction_req_processor_plugin.py

```diff
@@ -87,6 +87,7 @@ def bid(self, aucId, amount):
         return req
 
 
+@pytest.mark.skip(reason="old style plugin")
```

Comment:
What tests do we have for new style plugins?

Reply:
They are in the actual plugin packages

Reply:
But do we have (or going to have) any tests in indy-plenum that check that new pluging support works in principle? Are we going to have new-style plugins in plenum code (in next PRs) and corresponding tests ?

Reply:
And probably we should completely remove old-style plugins and tests code if we don't use/support them?

Reply:
We will not have any plugins in plenum codebase AFAIK. We might add one demo plugin

Reply:
I think it would be great to have tests (a demo plugin) for the plugins code we introduce here.
This is actually how it was before (we had old-style plugins, and had sample plugins to check that everything works). Let's not lose this.
BTW Are we going to have plugins in indy-node code base? Or all plugins will be only in separate projects (dependent on indy-node)?